multinet/downloaders/csv.py

```diff
@@ -0,0 +1,46 @@
+"""Multinet uploader for CSV files."""
+import csv
+from flasgger import swag_from
+from io import StringIO
+
+from .. import db, util
+from ..errors import NotFound
+
+from flask import Blueprint, make_response
+
+# Import types
+from typing import Any
+
+
+bp = Blueprint("download_csv", __name__)
+bp.before_request(util.require_db)
+
+
+@bp.route("/<workspace>/<table>", methods=["GET"])
```

Comment:
Jake and I did discuss this some (and he decided to publish a PR with this concept since it existed in code already) but we did both feel that using the existing API infrastructure would be good for this functionality. After looking at this I agree that the downloaders for tables should be found at `GET /{workspace}/tables/{table}/download` and those for graphs at `GET /{workspace}/graphs/{graph}/download`. I'm not sure if this is what you meant, @JackWilb, but I think it's the right choice. If/when we support multiple download file types, we can specify those with a `format` parameter.

Jake mentioned that our *uploaders* don't follow this format, and he has a point. In a future effort, we should probably find a way to distribute those `POST` endpoints into something like `POST /{workspace}/tables/{table}` with a `format` parameter specifying which type of file is coming. This might be difficult since some of our uploaders result in multiple tables being created, etc., so we'll have to determine what REST API best practices are for such a situation.

Reply:
Just for completeness: Jake also pointed out that doing these endpoints the way we have done the uploaders has one upside--the documentation is nice and explicit in the swagger page. To do it the more traditional way (i.e. how I'm suggested the downloaders be done) means that the documentation for the different supported file formats will all have to be crammed into the single `/download` endpoint's entry, but I am ok with that tradeoff.